utility/module_enable_actors_test.go

```diff
@@ -0,0 +1,99 @@
+package utility
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/runtime/configs"
+	"github.com/pokt-network/pocket/shared/modules"
+	mocks "github.com/pokt-network/pocket/shared/modules/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEnableActorModules(t *testing.T) {
+	tests := []struct {
+		name                 string
+		config               *configs.Config
+		expectedError        string
+		expectedNames        []string
+		expectedActorModules []modules.Module
+		expectedLogMessages  []string
+	}{
+		{
+			name: "servicer only",
+			config: &configs.Config{
+				Servicer: &configs.ServicerConfig{Enabled: true},
+			},
+			expectedNames: []string{"servicer"},
+		},
+		{
+			name: "fisherman only",
+			config: &configs.Config{
+				Fisherman: &configs.FishermanConfig{Enabled: true},
+			},
+			expectedNames: []string{"fisherman"},
+		},
+		{
+			name: "validator only",
+			config: &configs.Config{
+				Validator: &configs.ValidatorConfig{Enabled: true},
+			},
+			expectedNames: []string{"validator"},
+		},
+		{
+			name: "validator and servicer",
+			config: &configs.Config{
+				Validator: &configs.ValidatorConfig{Enabled: true},
+				Servicer:  &configs.ServicerConfig{Enabled: true},
+			},
+			expectedNames: []string{"validator", "servicer"},
+		},
+		{
+			name: "multiple actors not allowed",
+			config: &configs.Config{
+				Validator: &configs.ValidatorConfig{Enabled: true},
+				Fisherman: &configs.FishermanConfig{Enabled: true},
+			},
+			expectedError: ErrInvalidActorsEnabled,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			ctrl := gomock.NewController(t)
+			defer ctrl.Finish()
+
+			mockRunMgr := mocks.NewMockRuntimeMgr(ctrl)
+			cfg, err := configs.CreateTempConfig(test.config)
+			assert.NoError(t, err)
+
+			mockRunMgr.EXPECT().GetConfig().Return(cfg).AnyTimes()
+
+			bus, err := runtime.CreateBus(mockRunMgr)
+			assert.NoError(t, err)
+
+			// Call enableActorModules with the test config
+			m, err := Create(bus)
+
+			// Verify error output
+			if test.expectedError == "" {
+				assert.NoError(t, err)
```

Comment:
Use `require` instead of `assert`.

![Screenshot 2023-05-31 at 6 43 40 PM](https://github.com/pokt-network/pocket/assets/1892194/f3daee94-aba2-4102-abaf-24a4a78f2bb8)

https://github.com/stretchr/testify#require-package